src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -118,7 +118,9 @@ protected function findTemplate(Request $request, $format, $code, $debug)
         return new TemplateReference('TwigBundle', 'Exception', $debug ? 'exception_full' : $name, 'html', 'twig');
     }
 
-    // to be removed when the minimum required version of Twig is >= 2.0
+    /**
+     * To be removed when the minimum required version of Twig is >= 2.0.
```

Comment:
this is not suited for phpdoc. It is not documenting the method. It is a comment for people reading the source code.